src/app/retrospectCreate/RetrospectCreate.tsx

```diff
@@ -42,29 +42,43 @@ export function RetrospectCreate() {
     },
   } as const;
 
-  // const [mainInfo] = useAtom(mainInfoAtom);
-  // const [questions] = useAtom(questionsAtom);
-  const [date] = useAtom(dueDateAtom);
+  const navigate = useNavigate();
+  const locationState = useLocation().state as { spaceId: number };
+  const { spaceId } = locationState;
+  if (!spaceId) {
+    throw new Error("location으로부터 spaceId를 읽을 수 없습니다.");
+  }
+  const {
+    data: { formId },
+  } = useGetSpace(spaceId);
+
+  const [retroCreateData, _] = useAtom(retrospectCreateAtom);
+  const postRetrospectCreate = usePostRetrospectCreate(spaceId);
+
+  const steps = ["start", "mainInfo", "customTemplate", "dueDate"] as const;
+
+  const handleSubmit = () => {
+    postRetrospectCreate.mutate({
+      spaceId,
+      body: { ...retroCreateData },
+    });
+  };
 
   const { currentStep, goNext, goPrev, totalStepsCnt, currentStepIndex } = useMultiStepForm({
     steps,
     redirectPath: PATHS.completeRetrospectCreate(),
+    handleSubmit,
   });
 
   const conditionalIncrementPage = () => {
     if (currentStep === "dueDate") {
-      if (!date.date || !date.time) {
+      if (!retroCreateData.deadline) {
         return currentStepIndex - 1;
       }
     }
     return currentStepIndex;
   };
```

Comment:
오호! 이렇게도 표현할 수 있으려나요?!

```suggestion
const conditionalIncrementPage = () => {
  return currentStep === "dueDate" && !retroCreateData.deadline 
    ? currentStepIndex - 1 
    : currentStepIndex;
}
```

Reply:
오 훨씬 깔끔하다 오케이